app/model/user.js

```diff
@@ -209,6 +209,10 @@ userSchema.pre('save', function (next) {
 
 // sync with LDAP
 userSchema.pre('save', function (next) {
+  // disable ldap for dev
+  if (!process.env.NODE_ENV && !process.env.LDAP) {
```

Comment:
This is incorrect. You have to actually check the value of NODE_ENV and this should be logical OR, not logical AND.